### PR TITLE
chore(): update version axios

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: #8953

## What is the new behavior?
Update version axios [0.25.0](https://github.com/axios/axios/releases/tag/v0.25.0)

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
This PR, update version axios [0.25.0](https://github.com/axios/axios/releases/tag/v0.25.0)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
   },
   "dependencies": {
     "@nuxtjs/opencollective": "0.3.2",
-    "axios": "0.24.0",
+    "axios": "0.25.0",
```
Comment (kamilmysliwiec) [inline package.json:52]: Can you please also update Axios version in the common/package.json file?

Comment (Tony133) [reply package.json:52]: Yes sure 👍

Comment (Tony133) [reply package.json:52]: Done! 
I noticed that dependabot created the same dependency update a few minutes ago.
https://github.com/nestjs/nest/pull/8981

**Review by stephenwade — COMMENTED** [review]
**question:** Why are so many lines changed in package-lock.json? 33,000 lines seems like way too many for a package update.

**Comment by Tony133** [comment]
@stephenwade the package-lock.json was created as new, like when you first run with `npm install`

File: package.json (hunk 1 above)
Comment (Tony133) [reply package.json:52]: if you want to close this PR in favor of the dependabot one (created today), close it, no problem for me. 👍

**Pull request closed without merging**